### DISCUSSION TITLE
Unknown Fingerprint

### DISCUSSION THREAD

**Discussion by theologyco** [iterate-ch/cyberduck#15472] [Q&A]
I have been using Cyberduck on the same computer (MacBookPro wirh OS 10.13.6) for years and it has always worked perfetly until the last few days I have been getting an "unknown fingerprint:" message which begins "The fingerprint for the ssh-ed25519 . . ." then lists the fingerprint. Itgives me choices deny/allow/always. Two questions:

1. what does this mean? Have I inadvertently changed something? How do I fix it?

and 2. If I say allow can this stop the sftp from working or open a vulnerability?

cheers,

Jeffrey

**Comment by tatze96** [comment]
The "Unknown Fingerprint" message is usually a safety message that lets you know that Cyberduck doesn't know about this server in combination with the specific fingerprint. You can either allow the fingerprint if you are sure to connect to the right server or deny it if you are unsure. Denying the fingerprint will cause the cancellation of the connection process.

Cyberduck will remember the combination of server and fingerprint and won't show that window again for this server if you tick the checkbox *Always* before clicking on the *Allow* button.

**Comment by theologyco** [comment]
Thankyou Laura,

I will click on Always,

cheers,

Jeffrey
